### PR TITLE
nu: Add version 0.3.0

### PR DESCRIPTION
**Pull request by niheaven**
> Nu will look for the plugins in your PATH on startup. While nu will have some functionality without them, for full functionality you'll need to copy them into your path so they can be loaded.

### CODE REVIEW

File: bucket/nu.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "version": "0.3.0",
+    "license": "MIT",
+    "description": "A modern shell written in Rust.",
+    "url": "https://github.com/nushell/nushell/releases/download/0.3.0/nu_0_3_0_windows.zip",
+    "hash": "73f5ac08c3c3fe1f81ec45bcae95985068f859d3aceb1267e88b924317e227f7",
+    "extract_dir": "nushell_0_3_0",
+    "env_add_path": ".",
+    "checkver": {
+        "github": "https://github.com/nushell/nushell",
+        "regex": "/tag/([d.]+)"
```
Comment (Ash258) [inline bucket/nu.json]: ??
Why you need special regex for this?

Comment (niheaven) [reply bucket/nu.json]: Wo, yes, normal regex works.

**Comment by Ash258** [comment]
> you'll need to copy them into your path so they can be loaded.

Would not make sense to crate some folder **Plugins** folder inside application directory and then add normal bins and plugins folder to path.

**Comment by niheaven** [comment]
That sounds well, but we need to check if there're new ones in new release. I'll change it and check periodically.

**Pull request merged**